### PR TITLE
Sync config example in sonata_overview

### PR DESCRIPTION
**Pull request by mgeplf**
* to include the `populations` dictionary
* enable intersphinx
* fix tox compat

### CODE REVIEW

File: source/sonata_overview.rst [hunk 1]
```diff
@@ -43,17 +43,28 @@ A standard circuit config at |bbp| is :
         "nodes": [
           {
             "nodes_file": "$NETWORK_DIR/neurons.h5",
-            "node_types_file": "$NETWORK_DIR/node_types.csv"
+            "node_types_file": null,
```
Comment (jdcourcol) [inline source/sonata_overview.rst:46]: we can omit this since it is optional.
since the sentence says 'A standard circuit config at BBP is...

**Comment by jdcourcol** [comment]
I am good. I have no clue about the tox thing so I will let @GianlucaFicarelli comment on that.

**Review by GianlucaFicarelli — COMMENTED** [review]
LGTM, but I wonder why intersphinx is needed?

**Review by matz-e — APPROVED** [review]

**Comment by mgeplf** [comment]
> LGTM, but I wonder why intersphinx is needed?

I'm planning on linking snap doc to here, if it works out.

**Pull request merged**